pytests/tests/test_config.py

```diff
@@ -12,32 +12,30 @@
 
 @pytest.fixture(scope='module', autouse=True)
 def setup_test(utils):
-    utils.run([ 'cp', '/etc/yum.repos.d/photon.repo', '/etc/yum.repos.d/photon.repo.bak' ])
-    utils.run([ 'sed', '-i', 's/enabled=1/enabled=0/g', '/etc/yum.repos.d/photon.repo' ])
-    utils.run([ 'cp', '/etc/tdnf/tdnf.conf', '/etc/tdnf/mytdnf.conf' ])
-    utils.run([ 'sed', '-i', '/repodir/d', '/etc/tdnf/mytdnf.conf' ])
-    utils.run([ 'sed', '-i', '$ a repodir=/etc/myrepo', '/etc/tdnf/mytdnf.conf' ])
-    utils.run([ 'mkdir', '-p', '/etc/myrepo' ])
-    utils.run([ 'cp', '/etc/yum.repos.d/photon.repo', '/etc/myrepo/photon.repo' ])
-    utils.run([ 'sed', '-i', 's/enabled=0/enabled=1/g', '/etc/myrepo/photon.repo' ])
+    tdnf_conf = os.path.join(utils.config['repo_path'], 'tdnf.conf')
+    tdnf_repo = os.path.join(utils.tdnf_config.get('main', 'repodir'), 'photon-test.repo')
+    utils.run([ 'mkdir', '-p', '/tmp/myrepo' ])
+    utils.run([ 'cp', tdnf_conf, '/tmp/myrepo/mytdnf.conf' ])
+    utils.run([ 'sed', '-i', '/repodir/d', '/tmp/myrepo/mytdnf.conf' ])
+    utils.run([ 'sed', '-i', '$ a repodir=/tmp/myrepo', '/tmp/myrepo/mytdnf.conf' ])
+    utils.run([ 'cp', tdnf_repo, '/tmp/myrepo/photon.repo' ])
+    utils.run([ 'sed', '-i', 's/enabled=0/enabled=1/g', '/tmp/myrepo/photon.repo' ])
     yield
     teardown_test(utils)
 
 def teardown_test(utils):
-    utils.run([ 'cp', '/etc/yum.repos.d/photon.repo.bak', '/etc/yum.repos.d/photon.repo' ])
-    utils.run([ 'rm', '/etc/tdnf/mytdnf.conf' ])
     utils.run([ 'rm', '-rf', '/etc/myrepo' ])
```

Comment:
setup_test moved all this to /tmp/myrepo so this is not required

Reply:
Must have been `/tmp/myrepo`. Fixed